creator-node/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "target": "es5",
+    "module": "commonjs",
+    "lib": [
+      "esnext"
+    ],
+    "outDir": "./build",
+    "allowJs": true,
+    "downlevelIteration": true,
+    "strict": true,
+    "noFallthroughCasesInSwitch": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": false,
```

Comment:
we want this to be `true`, prob bc we import json files throughout our codebase

Reply:
i'm kind of surprised how this has worked so far with this setting false. i'll flip this flag